.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: julia
 os:
     - linux
-    - osx
```

Comment:
why is this line deleted?

Reply:
`osx` tests are added via `matrix.include`: https://github.com/JuliaPy/PyCall.jl/pull/602/files#diff-354f30a63fb0907d4ad57269548329e3R20

I think there would be duplicates if we left `osx` here.